twostep.php

```diff
@@ -0,0 +1,55 @@
+<?php
+/**
+ * Plugin Name: VIP Force Two Step
```

Comment:
Since the plugin UI uses "Two Factor" as well, we should probably just go with that.